java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1241,6 +1242,26 @@ public T zcard(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Adds all the elements to the HyperLogLog data structure stored and create a new structure if it
+     * is missing.
+     *
+     * <p>A command call without <code>elements</code> results in no operation being performed if the
+     * <code>key</code> already exists, or just the creation of the data structure if the <code>key
+     * </code> does not exist (in the latter case <code>1</code> is returned).
+     *
```

Comment:
if not a hyperloglog, an error is returned.

Reply:
I copied updated doc from the client interface